spec/paypal_express/remote/hpp_spec.rb

```diff
@@ -194,7 +198,7 @@
       validate_token(form)
 
       # Verify auth, capture and refund
-      authorize_capture_and_refund(kb_payment_id, payment_external_key, properties)
+      authorize_capture_and_refund(kb_payment_id, payment_external_key, properties, is_pending_payment_test ? payment_processor_account_id : default_processor_id)
```

Comment:
This doesn't seem correct: regardless whether a pending payment has been created, the same merchant account should be used to call `SetExpressCheckout` (during `buildFormDescriptor`) and `GetExpressCheckoutDetails` (during `add_payment_method`, `purchase`, etc.).